Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        private static void CheckAndResetWindowSize() // my consol is bugged, that is the only reason why this stuff exist.
+        {
+            if (Console.WindowHeight != 300 || Console.WindowWidth != 500)
+            {
+                Console.SetWindowSize(120, 40);
+            }
+        }
+        public static void Demo()
+        {
+            CheckAndResetWindowSize();
+            StuffThatExistTwice();
+            StuffThatExistTwice();
+        }
+        static void StuffThatExistTwice()
+        {
+            string name = Name();
+            double age = Age();
+            double weight = Weight();
+            double height = Height();
+            double BMI = Math.Round(weight / Math.Pow(height/100,2),2);
+            Console.WriteLine($"{name} is {age} old, their weight is {weight}kg and their height is {height}cm\n Their BMI index is {BMI}");//their is used instead of his/her.
+
+        }
+        static void Text(string text)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename the parameter 'text' so that it does not duplicate the method name.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4905569)